crates/egui/src/lib.rs

```diff
@@ -507,7 +507,7 @@ pub fn warn_if_debug_build(ui: &mut crate::Ui) {
 /// ```
 #[macro_export]
 macro_rules! include_image {
-    ($path: literal) => {
+    ($path:expr $(,)?) => {
```

Comment:
why the `$(,)?` though?

Reply:
I just copied it from [include_bytes](https://doc.rust-lang.org/src/core/macros/mod.rs.html#1393) to be consistent. I'm not sure what it is for.

Reply:
🤷